Gaijinizer patch/CommonEvents_643.txt

```diff
@@ -1,208 +1,266 @@
 ﻿> GAIJINIZER PATCH FILE
 > BEGIN STRING
 \\N[1]は、モルモーに犯されてしまった！！
-> CONTEXT: CommonEvents_643/13/Dialogue < UNTRANSLATED
-> CONTEXT: Troops/319/pages/0/37/Dialogue < UNTRANSLATED
-
+> CONTEXT: CommonEvents_643/13/Dialogue
+> CONTEXT: Troops/319/pages/0/37/Dialogue
+\\N[1] is being raped by Mormo!!#Ron
 > END STRING
 
 > BEGIN STRING
 ふふふ…\\N[1]、かわいい…♡
 
 そんなトロンとした表情しちゃって…♡
-> CONTEXT: CommonEvents_643/40/Dialogue/enemy25 < UNTRANSLATED
+> CONTEXT: CommonEvents_643/40/Dialogue/enemy25
+Fufufu, you're so cute. ♡
 
+You've got such a slack look on your face.#Ron
 > END STRING
 
 > BEGIN STRING
 普段は首筋からがメインだから滅多にアピールしないけど
 
 こっちに意識を集中させての正統派な搾精も得意なんだ♡
-> CONTEXT: CommonEvents_643/44/Dialogue/enemy25 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_643/44/Dialogue/enemy25
+I usually go for the neck so I don't often show
+myself off like this. But this kind of orthodox
+squeezing method where I get you to focus on me
+like this is also my specialty. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 ボクは夢魔だから、サキュバスと同じく
 瞬時に相手が気持ちいいと思う『ツボ』が分かるし…
 
 攻め方も、加減も、ぜーんぶ相手の理想通り…
-> CONTEXT: CommonEvents_643/48/Dialogue/enemy25 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_643/48/Dialogue/enemy25
+Like every other succubus, my aim is to make my
+partner feel good straight away. So the way I
+attack, the intensity I use, it's all based on
+the ideals of my partner.#RonIffyWording
```

Comment:
![image](https://user-images.githubusercontent.com/48231102/119388358-6a99c000-bc87-11eb-93bd-f1851722e06f.png)

Hmm... is this interpretation valid? As in she knows exactly what spots to hit immediately?

Reply:
Yeah, you could say my translation was somewhat "abstracted".

Reply:
Okay, let's go with this then:
```
Like all succubus, I know exactly what makes you
feel good... So the way I attack, the intensity
I use... It's all based on exactly what my
partner wants...
```